contrib/repairStatus.py

```diff
@@ -0,0 +1,199 @@
+from __future__ import print_function
```

Comment:
another good `__future__` import is `unicode_literals`, which makes all of your quoted strings be really actual strings, and not bytes-masquerading-as-strings (which is what "str" is in python 2).  This might necessitate fixing up some things in your code, but those are almost always good things to fix. Given that 99% of your quoted strings here are in JSON structures, and JSON structures are really supposed to be unicode anyway, this is probably the right way to go.

(You can still type bytes literals with the b-prefix syntax, like `b"foo"`.)